src/layouts/ReviewRequest/Dashboard.tsx

```diff
@@ -13,127 +13,271 @@ import {
   MenuDropdownItem,
 } from "components/MenuDropdownButton"
-import { useState } from "react"
+import { useEffect, useRef, useState } from "react"
 import { BiLink, BiPlus } from "react-icons/bi"
+import { useParams } from "react-router-dom"
 
-import { extractInitials, getDateTimeFromUnixTime } from "utils"
+import { useRoleContext } from "contexts/RoleContext"
 
-import { RequestOverview, EditedItemProps } from "./components/RequestOverview"
+import { useApproveReviewRequest } from "hooks/reviewHooks/useApproveReviewRequest"
+import { useGetCollaborators } from "hooks/reviewHooks/useGetCollaborators"
+import { useGetReviewRequest } from "hooks/reviewHooks/useGetReviewRequest"
+import { useMergeReviewRequest } from "hooks/reviewHooks/useMergeReviewRequest"
+import useRedirectHook from "hooks/useRedirectHook"
 
-export interface ReviewRequestDashboardProps {
-  reviewUrl: string
-  title: string
-  requestor: string
-  reviewers: string[]
-  reviewRequestedTime: Date
-  changedItems: EditedItemProps[]
+import { getAxiosErrorMessage } from "utils/axios"
+
+import { ReviewRequestStatus } from "types/reviewRequest"
+import { extractInitials, getDateTimeFromUnixTime, useErrorToast } from "utils"
+
+import { SiteViewHeader } from "../layouts/SiteViewLayout/SiteViewHeader"
+
+import { CancelRequestModal, ManageReviewerModal } from "./components"
+import { ApprovedModal } from "./components/ApprovedModal"
+import { PublishedModal } from "./components/PublishedModal"
+import { RequestOverview } from "./components/RequestOverview"
+
+export const ReviewRequestDashboard = (): JSX.Element => {
+  const { role } = useRoleContext()
+  const { siteName, reviewId } = useParams<{
+    siteName: string
+    reviewId: string
+  }>()
+  const { setRedirectToPage } = useRedirectHook()
+  // TODO!: Refactor so that loading is not a concern here
+  const { data, isLoading: isGetReviewRequestLoading } = useGetReviewRequest(
+    siteName,
+    parseInt(reviewId, 10)
+  )
+  const { data: collaborators, isLoading, isError } = useGetCollaborators(
+    siteName
+  )
+  // TODO!: redirect to /sites if cannot parse reviewId as string
+  const { onCopy, hasCopied } = useClipboard(data?.reviewUrl || "")
+
+  const reviewStatus = data?.status
+  const isApproved = reviewStatus === ReviewRequestStatus.APPROVED
+
+  useEffect(() => {
+    if (
+      reviewStatus === ReviewRequestStatus.CLOSED ||
+      reviewStatus === ReviewRequestStatus.MERGED
+    ) {
+      setRedirectToPage(`/sites/${siteName}/dashboard`)
+    }
+  }, [reviewStatus, setRedirectToPage, siteName])
+
+  return (
+    <>
+      <Box bg="white" w="100%" h="100vh">
+        <VStack
+          bg="blue.50"
+          pl="9.25rem"
+          pr="2rem"
+          pt="2rem"
+          pb="1.5rem"
+          spacing="0.75rem"
+          align="flex-start"
+        >
+          <Flex w="100%">
+            <HStack spacing="0.25rem">
+              <Text textStyle="h5">{data?.title || ""}</Text>
```

Comment:
Minor nit: Can we switch to using a skeleton instead? Also display something like "Unable to retrieve data" if the review request could not be loaded.

Reply:
oh yes - i'm aware of this and i haven't created a tracking issue yet AHHA

Reply:
tracked by #1126